### DISCUSSION TITLE
General Assistance: How is the caddy handling in Kubernetes?

### DISCUSSION THREAD

**Discussion by sevko21** [SecurityRiskAdvisors/VECTR#279] [General]
With Vectr 9.2.1 caddy is part of the deployment. How can this be handled within a Kubernetes Cluster?
Our current release 9.0 is running well on the cluster but dont know how to upgrade to 9.2.1 with tls termination not going over my ingress.

**Comment by SRAPSpencer** [comment]
We don't offer support Kubernetes for VECTR community edition. We know users have successfully deployed VECTR to such environments and others are welcome to comment if they have suggestions if they would like. I will mark support for such deployments as a feature request.

**Comment by p-b--** [comment]
Flipping this to a discussion for other community members to comment.
